src/hooks/useLoginApi.ts

```diff
@@ -80,23 +80,16 @@ const useLogoutApi = () => {
     []
   );
 
-  const callbackSuccess = useCallback((localStorageTheme?: string | null) => {
+  const callbackSuccess = useCallback(() => {
     cookies.remove(`${process.env.REACT_APP_AUTH_COOKIE}`);
+    localStorage.removeItem(`${process.env.REACT_APP_THEME_KEY}`);
```

Comment:
Could we use `clear()` to make this more future-proof in case someone starts storing more in local storage but forgets to update this callback?

```suggestion
    localStorage.clear();
```